IBAnimatable/IBAnimatable tvOSTests/IBAnimatable_tvOSTests.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  IBAnimatable_tvOSTests.swift
+//  IBAnimatable tvOSTests
+//
+//  Created by Steven on 6/9/17.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+import XCTest
+@testable import IBAnimatable
+
+class IBAnimatable_tvOSTests: XCTestCase {
```

Comment:
No, I can do the same thing as the source target and wrap the tests that are only available on iOS with `#if os(IOS)`.